mParticle-Pilgrim.podspec

```diff
@@ -0,0 +1,20 @@
+Pod::Spec.new do |s|
+    s.name             = "mParticle-Pilgrim"
+    s.version          = "7.7.3"
+    s.summary          = "Pilgrim integration for mParticle"
+
+    s.description      = <<-DESC
+                       This is the Pilgrim integration for mParticle.
+                       DESC
+
+    s.homepage         = "https://www.mparticle.com"
+    s.license          = { :type => 'Apache 2.0', :file => 'LICENSE' }
+    s.author           = { "mParticle" => "support@mparticle.com" }
+    s.source           = { :path => "/Users/creyes/Workspace/mparticle-apple-integration-example" }
```

Comment:
Can you update this when you get a chance? Guessing you changed to make pod lib lint succeed.

Reply:
should this source point to our github repository then?

Reply:
Hey sorry I should have been more precise, the kit will be published in the mparticle-integrations org so this should look like the following:

```
s.source           = { :git => "https://github.com/mparticle-integrations/mparticle-apple-integration-pilgrim.git", :tag => s.version.to_s }
```

Reply:
awesome thanks!